File: temp/code.py
import json
import os
x = 24
print("Once again thanks")

print("x is now "+str(x)+", so I think it worked")

print("Also multi-line statements like if statements and while/for loops shoudl work now, so let's test that")

print("I'll print if x is equal to 24")

if x == 24:

  print("Yes, x is equal to 24")

else:

  print("No, x is not equal to 24")
dout = {}
dout['x'] = x
with open(os.path.dirname(os.path.abspath(__file__))+'/out.json', 'w') as f:
	json.dump(dout, f)
	f.close()

File: temp/code.js
var x = 12;
console.log("Thank you, so let's get it over with")

console.log("x is now set to "+x)

console.log("I'll add another 12 and send it to you, python\n")

x += 12

;(() => {
const fs = require('fs');
const path = require('path');
let dout = {};
dout['x'] = x;
fs.writeFileSync(path.join(__dirname, 'out.json'), JSON.stringify(dout));
})()